### DISCUSSION TITLE
How to create an information prompt window in the macro script that closes after 2 seconds?

### DISCUSSION THREAD

**Discussion by fengzyf** [WinEdt-Team/WinEdt-Support#84] [1.1 General Help and Discussion]
Dear Alex,

Currently, the prompt window created using the macro `Prompt` does not support automatic closing. Is there a way to create an information prompt window that displays for only 2 seconds?

Thank you!

**Comment by Alex-Simonic** [comment]
Currently, this cannot be done. However, the `Prompt` macro allows an extra optional integer parameter for the next version. If non-zero then the message automatically closes after the specified time in msec. The following will do the trick in 2 seconds:

`Prompt('Test...',0,1,'','','',2000);
`
